modules/awesomeBooks.js

```diff
@@ -0,0 +1,51 @@
+import Book from './book.js';
+
+export default class AwesomeBooks {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+    this.bookList = document.getElementById('bookList');
+    this.addBookForm = document.getElementById('bookContent');
+    this.addBookForm.addEventListener('submit', this.submitBookForm.bind(this));
+    this.displayBooks();
+  }
+
+  displayBooks() {
+    this.bookList.innerHTML = '';
+
+    this.books.forEach((book, index) => {
+      const listItem = document.createElement('li');
+      listItem.className = 'table-row';
+      listItem.innerHTML = `
+      <span class="table-cell">"${book.title}" by ${book.author}</span>
+      <span class="table-cell"><button class="remove-btn">Remove</button></span>
+      `;
+      const removeButton = listItem.querySelector('.remove-btn');
+      removeButton.addEventListener('click', () => {
+        this.removeBook(index);
+      });
+      this.bookList.appendChild(listItem);
+    });
+  }
```

Comment:
Nice Job so far👍😉😉, I suggest you kindly refactor your code to make use of the `es6` arrow function as this is a requirement for this milestone😉 apart from that, the arrow function has fewer complications unlike the regular functions, hence it's easier to make use of😉.

<img width="574" alt="image" src="https://github.com/rubydevi/book-collection-es6/assets/64643995/91231b75-6503-4f17-bc25-472d5a91bc44">

```suggestion
  displayBooks = () => {
    this.bookList.innerHTML = '';

    this.books.forEach((book, index) => {
      const listItem = document.createElement('li');
      listItem.className = 'table-row';
      listItem.innerHTML = `
      <span class="table-cell">"${book.title}" by ${book.author}</span>
      <span class="table-cell"><button class="remove-btn">Remove</button></span>
      `;
      const removeButton = listItem.querySelector('.remove-btn');
      removeButton.addEventListener('click', () => {
        this.removeBook(index);
      });
      this.bookList.appendChild(listItem);
    });
  }
```

Reply:
Thank you so much for pointing this out, I totally miss to refactor that part of the project @Ibizugbe